app/Assets/Helpers.php

```diff
@@ -261,4 +262,23 @@ public function censor(string $string, float $percentOfClear = 0.5): string
 
 		return substr_replace($string, $replacement, $start, $censored_length);
 	}
+
+	/**
+	 * Format exception trace as text.
+	 *
+	 * @param \Exception $e
+	 *
+	 * @return string
+	 *
+	 * @codeCoverageIgnore
+	 */
+	public function exceptionTraceToText(\Exception $e): string
```

Comment:
Not used in the PR, but quite useful when debugging:
```php
dd($var, Helpers::exceptionTraceToText(new Exception))
```
Will dump the value of $var in a 500 and followed by the trace of where the dump happened.
The front-end is already taking care of making it a nice visual :)